src/components/PdfLoader.js

```diff
@@ -4,7 +4,7 @@ import React, { Component } from "react";
 
 import type { T_PDFJS, T_PDFJS_Document } from "../types";
 
-import { PDFJS } from "pdfjs-dist";
+import pdfjs from 'pdfjs-dist/webpack';
```

Comment:
`webpack` import needed for the worker

Reply:
Seems this import doesn’t work with webpack 4.

Reply:
which error did you get?

Reply:
```
DeprecationWarning: Tapable.apply is deprecated. Call apply on the plugin directly instead
```